draft-ietf-taps-interface.md

```diff
@@ -1121,22 +1121,22 @@ not wish to apply a time constraint on the transmission of the Message, but it d
 reliable delivery; reliability is adjustable per Message via the "Reliable Data Transfer (Message)"
 property (see {{msg-reliable-message}}). The type and units of Lifetime are implementation-specific.
 
-### Niceness {#msg-niceness}
+### Priority {#msg-priority}
 
 Type:
 : Integer (non-negative)
 
-This property represents an unbounded hierarchy of priorities.
+This property represents a hierarchy of priorities.
 It can specify the priority of a Message, relative to other Messages sent over the
 same Connection.
 
-A Message with Niceness 0 will yield to a Message with Niceness 1, which will
-yield to a Message with Niceness 2, and so on. Niceness may be used as a
+A Message with Priority 0 will yield to a Message with Priority 1, which will
+yield to a Message with Priority 2, and so on. Priorities may be used as a
 sender-side scheduling construct only, or be used to specify priorities on the
 wire for Protocol Stacks supporting prioritization.
```

Comment:
should we say something here about what the max prio is?

Reply:
I think not - this should be application dependent. Giving a max of 10 or 100 or whatever they want. I only removed the statement to say that it's unbounded - with Niceness or Priority, this could be done, but is that better or worse than limiting it? I don't know. Let them decide.

Reply:
I see no reason to specify a concrete "max prio" – but perhaps we should define a read-only property to query it

Reply:
okay, we'll just leave this as undefined (and a nonportable value in the abstract API, which is also fine)